MyStore/src/include/GeneratorData.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <thread>
+#include <mutex>
+#include <conio.h>
+#include <chrono>

```

Comment:
These includes are not necessary in this header. Move them to appropriate .cpp files. As a rule of thumb include only required files in .h files, other ones move to .cpp files. Every additional include in MyHeader.h file will affect compilation time of modules that include MyHeader.h.